### ISSUE TITLE
Проект

### ISSUE THREAD

**Issue by AndreyTyshevich** [AndreyTyshevich/ArtilleryGame#1]
b9f7a703f165a605b736a629dfca106c733a09cd

**Comment by AndreyTyshevich** [comment]
@antosha417

**Comment by mishakrengauz** [comment]
@AndreyTyshevich Я загрузил программу, рисующую меню, и картинку, к нему прилагающуюся

**Comment by AndreyTyshevich** [comment]
Задания на завтра:
Галя - заменить снаряды на картинки, вставить фон и прописать отображение здоровья игроков.
Миша - прописать смену ветра и отрисовки флага, переделать меню, прописать нанесение урона и окончание игры

**Comment by StGalya** [comment]
после изменения основного кода картинки перестали адекватно вставляться. и я не знаю, что с этим делать.
419daf5903884e2758cb899c04749a1c9a2dc72a
@AndreyTyshevich

**Comment by StGalya** [comment]
так, ошибочка вышла. добавила картинки в game. cannot не исправляла
